docs/conf.py

```diff
@@ -89,3 +102,32 @@
     "https://docs.q-ctrl.com/assets/js/readthedocs.js",
 ]
 html_logo = "_static/logo.svg"
+
+
+public_apis = qctrlopencontrols.__all__
+
+autosummary_context = {
```

Comment:
Would you mind adding some brief comments explaining what each of these blocks is for, for the reader who's unfamiliar with autosummary (specifically `autosummary_context`, the `autosummary_filename_map`, and the class members handling)?